src/main/scala/org/ergoplatform/nodeView/wallet/AugWalletTransaction.scala

```diff
@@ -33,7 +34,7 @@ object AugWalletTransaction {
       "id" -> obj.wtx.tx.id.asJson,
       "inputs" -> obj.wtx.tx.inputs.asJson,
       "dataInputs" -> obj.wtx.tx.dataInputs.asJson,
-      "outputs" -> obj.wtx.tx.outputs.asJson,
+      "outputs" -> obj.wtx.tx.outputs.toSeq.asJson(Encoder.encodeSeq(enc)),
```

Comment:
same here, why this is necessary?

Reply:
@aslesarenko `enc` is brought in scope above as custom `ErgoBox` encoder. I wanted to emphisize that we're using it and not the one from `JsonCodecs`.

Reply:
Let's leave this as a comment in the code

Reply:
Added comment.